docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -12460,6 +12460,122 @@ CHAR(20). This expression uses the TRIM function to return the value
 TRIM(first_name) || ' ' || TRIM (last_name)
 ```
 
+[[trunc_function]]
+== TRUNC Function
+
+The TRUNC (TRUNCATE) function truncates `_numeric-expression_` on an optional `_decimal-place_` decimal position(s), that is, it rounds down `_numeric-expression_` to the nearest (or equal) integer closer to zero *without rounding*.
+
+TRUNC is a Trafodion SQL extension.
+
+`TRUNC (_numeric-expression_[, _decimal-place_])`
+
+[[syntax_descriptions_of_trunc]]
+=== Syntax Descriptions of TRUNC
+
+* `_numeric-expression_`
+
++
+is an SQL numeric expression to be truncated.
+
++
+For more information, see <<numeric_value_expressions,Numeric Value Expressions>>.
+
+* `_decimal-place_`
+
++
+is an integer specifying the precision of the truncation.
+
+NOTE: `_numeric-expression_` and `_decimal-place_` cannot be NULL.
+
+NOTE: Currently, `_decimal-place_` can be zero or a positive integer, while *negative integer* is *not supported*.
```

Comment:
Ah, this makes it easier. We don't have to worry about the negative case for the second argument.

Reply:
For most databases, TRUNC(123,-1) gives 120, while Trafodion doesn't support negative _decimal-place_.